ya.conf

```diff
@@ -8,6 +8,7 @@ USE_ASMLIB="no"
 USE_ARCADIA_CUDA="no"
 CATBOOST_OPENSOURCE="yes"
 CFLAGS="-DCATBOOST_OPENSOURCE=yes"
+CXXSFLAGS="-DCATBOOST_OPENSOURCE=yes"
```

Comment:
Is it a typo (CXX**S**FLAGS)?

Reply:
Yes sorry in the progress of creating a clean branch for the pull request I messed up the spelling. I’ll update in the weekend

Reply:
I've pushed a correction